app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,10 @@
   <title><%= content_for?(:html_title) ? yield(:html_title) : "Walltaker" %></title>
   <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1">
   <meta name="rating" content="adult"/>
+  <% mascot = current_user&.mascot || 'ki' %>
```

Comment:
I didn't wanna leave two separate places where the mascot display logic defaults to Ki, so I put that here--but it doesn't feel like it belongs here. Would this line be better at the top of the file?